File: leetcode/1051-height-checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        expected = sorted(heights)
        diff_count = 0
        for h, exp in zip(heights, expected):
            if h != exp:
                diff_count+=1
        return diff_count


class SolutionOneliner:
    def heightChecker(self, heights: List[int]) -> int:
        return sum(1 if h != exp else 0 for  h, exp in zip(heights, sorted(heights)))

File: src/1051-height-checker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
var heightChecker = function (heights) {
    let expected = [...heights].sort((a, b) => a - b)
    let result = 0
    for (let i = 0; i <= heights.length - 1; i++) {

        if (heights[i]!= expected[i]) {
            result=result +1
        }
    } 
    return result

};